test/kb_SPAdes_server_test.py

```diff
@@ -700,7 +700,7 @@ def test_metagenome(self):
                }],
              'md5': '08d0b92ce7c0a5e346b3077436edaa42',
              'fasta_md5': 'ca42754da16f76159db91ef986f4d276'
-             }, dna_source='metagenomic')
+             }, min_contig_len=500, dna_source='metagenomic')
```

Comment:
Wait a sec - why is this test passing? Surely changing the min contig length should have changed the output?